MC/config/PWGDQ/EvtGen/GeneratorEvtGen.C

```diff
@@ -42,6 +42,7 @@ enum DecayModeEvt { kEvtAll = 0,
                     kEvtElectronEM,
                     kEvtDiElectronEM,
                     kEvtGammaEM,
+                    kEvtBPsiJpsiPiPi,
```

Comment:
maybe lets improve this name to be more exact. Can you use something like kEvtBtoPsi2SToJpsiPiPi ?